github/github.go

```diff
@@ -110,6 +107,9 @@ const (
 	mediaTypeInteractionRestrictionsPreview = "application/vnd.github.sombra-preview+json"
 
 	// https://developer.github.com/changes/2019-03-14-enabling-disabling-pages/
+	// https://developer.github.com/changes/2019-02-14-draft-pull-requests/
```

Comment:
I don't think you meant to add this one back, did you?

Reply:
No, I didn't. Sorry, my bad